src/test/java/com/commafeed/backend/service/FeedUpdateServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.commafeed.backend.service;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+import java.util.List;
+import org.junit.Test;
+
+import com.commafeed.backend.model.Feed;
+import com.commafeed.backend.model.FeedEntry;
+import com.commafeed.backend.model.FeedSubscription;
+
+import junit.framework.Assert;
+
+/**
+* This class will test methods of the class FeedUpdateService.java
+*/
+public class FeedUpdateServiceTest {
+
+	@Test
+	public void testAddEntry() {
+		//Mocking dependencies
+		FeedUpdateService fus = mock(FeedUpdateService.class);
+		
+		Feed feed = mock(Feed.class);
+		FeedEntry feedEntry = mock(FeedEntry.class);
+		List<FeedSubscription> subscriptions = (List<FeedSubscription>) mock(FeedSubscription.class);
+		
+		feedEntry = new 
+		
+		//Expecting return false when feedEntry is null
+		when(fus.addEntry(feed, feedEntry, subscriptions)).thenReturn(false);
```

Comment:
This makes no sense. You mock the class that you are actually testing. You are saying: when this mock I created calls this method and those parameters, then return false.
Next line, you are saying assert that when this mock I created calls the method and the params mentioned above, it returns false. 
In short, you are testing mockito, not the addEntry method from FeedUpdateService.